### PR TITLE
fix: fix outdated & incorrect code example

### PR DESCRIPTION
**Pull request by linbudu599**
Closes: #293

- The `skip type definition` and `result return from getQuestions.ts` problem mentioned in the issue is my carelessness, content which generated from the latest version of BlitzJS works normally.

I found some other issues and  fix it, including: 
- `<Link />` component props:
- `include` options in getQuestion.ts
- keep up with latest generated contents

### CODE REVIEW

File: docs/tutorial.mdx [hunk 1]
```diff
@@ -513,32 +509,46 @@ export default async function getQuestion({where /* include */}: GetQuestionInpu
 import { Ctx } from "blitz"
 import db, { Prisma } from "db"
 
-type GetQuestionsInput = Pick<Prisma.FindManyQuestionArgs, "where" | "orderBy" | "cursor" | "take" | "skip">
+type GetQuestionsInput = Pick<Prisma.FindManyQuestionArgs, "where" | "orderBy" | "skip" | "take">
 
 export default async function getQuestions(
-  {where, orderBy, cursor, take, skip}: GetQuestionsInput,
-  ctx: Ctx,
+  { where, orderBy, skip = 0, take }: GetQuestionsInput,
```
Comment (linbudu599) [inline docs/tutorial.mdx:515]: It seems that I just copy the code when walking through the tutorials :)
so I just change the content to the latest generated code.

File: docs/tutorial.mdx [hunk 2]
```diff
@@ -513,32 +509,46 @@ export default async function getQuestion({where /* include */}: GetQuestionInpu
...
 
-Now hop back to our main questions page in your editor, and we can list the choices of each question. Add this code beneath the `Link` in our `QuestionsList`:
+Now hop back to our main questions page in your editor, and we can list the choices of each question.
+
+Add `include` option in `app/questions/queries/getQuestion.ts` to fetch `Question` data with its choices info:
+
+```
+const question = await db.question.findFirst({
+  where,
+  include: { choices: true }, // highlight-line
```
Comment (linbudu599) [inline docs/tutorial.mdx:547]: We cannot use `question.choices` if we don't include choices relations in query operation.

File: docs/tutorial.mdx [hunk 3]
```diff
@@ -640,7 +641,7 @@ export const Question = () => {
         where: {id},
         data: {votes: votes + 1},
       })
-      refetch()
```
Comment (flybayer) [inline docs/tutorial.mdx:643]: Sweet, thank you!!

Looks good except we need to keep this `refetch()` so the query get's updated after the mutation.

Comment (linbudu599) [reply docs/tutorial.mdx:643]: 🧐done

**Comment by flybayer** [comment]
Perfect!

**Pull request merged**